styles/roblox/catppuccin.user.css

```
@@ -0,0 +1,200 @@
+          background-color: @surface0
+      }
+
+      .chat-windows-header, .create-chat-container .details-header-container, .select-friends-container .details-header-container, .conversation-title-container .conversation-title-input, .btn-generic-more-sm:focus, .btn-generic-more-sm:hover, a.btn-generic-more-sm:focus, a.btn-generic-more-sm:hover, button.btn-generic-more-sm:focus, button.btn-generic-more-sm:hover, .btn-generic-edit-sm:hover, .chat-main .chat-body, .chat-search, .payment-method-label-clickable .payment-method-label:hover, .trade-row.selected, .checkbox input[type=checkbox]:checked+label:before, .chat-container .chat-main .chat-header, .rbx-select-group .rbx-select option:checked,  .rbx-select-group .rbx-select option:hover,  .rbx-select-group .select-option option:checked, .dar-theme .rbx-select-group .select-option option:hover,  .select-group .rbx-select option:checked,  .select-group .rbx-select option:hover, .select-group .select-option option:checked,  .select-group .select-option option:hover, .avatar-thumbnail .toggle-three-dee, .btr-explorer-item:not(.selected):hover, .btr-properties div.btr-property-group, .btr-explorer-filter input.btr-explorer-filter-input, .btr-explorer-header, .btr-properties-header, .btr-settings-footer, .btr-settings-header, #header .new-dropdown-menu .btr-fastsearch-anchor:hover, .dropdown-menu li:focus,  .dropdown-menu li:hover, .btr-thumb-btn-container .btr-body-popup .btr-body-outfits .btr-body-outfit-icon, .profile-avatar-left .enable-three-dee, .btr-bundle-animations .btr-bundle-btn, body .btr-bundle-animations .btr-bundle-btn-alt, .btr-thumb-btn-container .btr-thumb-btn{
+          background: @surface1;
+      }
+      .btr-explorer-item{
+          color: @text;
+      }
+      .btr-explorer-header, .btr-properties-header{
+          color: @text;
+      }
+      .btr-property-value input[type="text"], .btr-explorer-parent .btr-explorer div, .btr-explorer-parent .btr-properties div, .btr-explorer-filter input.btr-explorer-filter-input{
+          color: @text;
+      }
+      cashlabel{
+          color: @accent-color !important;;
+      }
+      .rbx-game-server-item .btn-control-xs, .rbx-friends-game-server-item .btn-control-xs, .rbx-private-server-item .btn-control-xs{
+          background: transparent;
+      }
+      .rbx-scrollbar-dragger, .rbx-scrollbar .mCSB_scrollTools .mCSB_dragger .mCSB_dragger_bar, .chat-search .chat-search-input, .notification-stream-header, .btr-properties .btr-property-group:hover, .btr-explorer-header .btr-dropdown-container .btr-dropdown-menu, .btr-explorer-header .btr-dropdown-container .btr-dropdown-btn, .rbx-header .navbar-search .navbar-search-option.new-selected{
+          background: @surface2
+      }
+  }
+
+  body.light-theme {
+    #catppuccin(@lightFlavor, @accentColor);
+background-color: @base;
+  }
+body.dark-theme {
+    #catppuccin(@darkFlavor, @accentColor);
+background-color: @base;
+  }
```

You can deduplicate the `background-color: @base;` from here and moving it into the `#catppuccin()` minix

```suggestion
  body.light-theme {
    #catppuccin(@lightFlavor, @accentColor);
  }
body.dark-theme {
    #catppuccin(@darkFlavor, @accentColor);
 }
```